pkg/sql/sqlstats/insights/integration/integration_test: TestInsightsIntegrationForContention failed

pkg/sql/sqlstats/insights/integration/integration_test.TestInsightsIntegrationForContention [failed](https://mesolite.cluster.engflow.com/invocations/default/2410d4af-10c7-402b-a5d1-4b15132be87a?testReportRun=2&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc3FsL3NxbHN0YXRzL2luc2lnaHRzL2ludGVncmF0aW9uOmludGVncmF0aW9uX3Rlc3Q=) on release-25.3 @ [9e8fbe32876f25ff52541edbc2f8fa6cc1d97593](https://github.com/cockroachdb/cockroach/commits/9e8fbe32876f25ff52541edbc2f8fa6cc1d97593):


```
=== RUN   TestInsightsIntegrationForContention
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestInsightsIntegrationForContention650729233
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:156: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    insights_test.go:831: condition failed to evaluate within 3m45s: from insights_test.go:839: waiting_txn not found
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestInsightsIntegrationForContention650729233
--- FAIL: TestInsightsIntegrationForContention (240.53s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=2</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/obs-prs
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestInsightsIntegrationForContention.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-55648

--- a/pkg/sql/sqlstats/insights/integration/insights_test.go
+++ b/pkg/sql/sqlstats/insights/integration/insights_test.go
@@ -780,6 +780,8 @@ func TestInsightsIntegrationForContention(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
+	skip.UnderDuress(t)
+
 	// Start the cluster. (One node is sufficient; the outliers system is currently in-memory only.)
 	ctx := context.Background()
 	settings := cluster.MakeTestingClusterSettings()

